test/automerge-integration.jl

```diff
@@ -283,6 +315,20 @@ hello_world_commit2 = "57b0aec49622faa962c6752d4bc39a62b91fe37c"
                             master_branch=master,
                             master_branch_is_default_branch=false,
                         )
+                        merge()
+                        if create_blocking_comment
+                            # Check we have the blocked label
+                            labels = GitHub.labels(repo, pr)
+                            @test AutoMerge.has_label(labels, AutoMerge.BLOCKED_LABEL)
+                            # Delete the comment & rerun
+                            GitHub.delete_comment(repo, blocking_comment; auth=auth)
+                            sleep(1)
+                            merge()
+                            sleep(1)
+                            # Check we no longer have the blocked label
+                            labels = GitHub.labels(repo, pr)
+                            @test !AutoMerge.has_label(labels, AutoMerge.BLOCKED_LABEL)
```

Comment:
this test is failing, but I'm not sure why. The label does seem to be deleted (https://github.com/bcbi-test/automerge-integration-test-repo/pull/11079)

CI:https://github.com/JuliaRegistries/RegistryCI.jl/actions/runs/11469400064/job/31916569056#step:8:1307